services/storage/src/simcore_service_storage/modules/celery/example_tasks.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
```

Comment:
this module should not exist. Why is it called example tasks? If you are not using it remove it, otherwise maybe call ti exporting_tasys and leave a note that the functions need to be repalced